auth0/v3/management/organizations.py

```diff
@@ -0,0 +1,349 @@
+
+    def get_organization_connection(self, id, connection_id):
+        """Retrieves an organization connection by its ID.
+
+        Args:
+           id (str): the ID of the organization.
+
+           connection_id (str): the ID of the connection.
+
+        See: https://auth0.com/docs/api/management/v2#!/Clients/get_clients
+        """
+        params = {}
+
+        return self.client.get(self._url(id, 'enabled_connections', connection_id), params=params)
+
+    def create_organization_connection(self, id, body):
+        """Adds a connection to an organization.
+
+        Args:
+           id (str): the ID of the organization.
+
+           body (dict): Attributes for the connection to add.
+
+        See: https://auth0.com/docs/api/v2#!/Clients/post_clients
+        """
+
+        return self.client.post(self._url(id, 'enabled_connections'), data=body)
+    
+    def update_organization_connection(self, id, connection_id, body):
+        """Modifies an organization.
+
+        Args:
+           id (str): the ID of the organization.
+
+           connection_id (str): the ID of the connection to update.
+
+           body (dict): Attributes to modify.
+
+        See: https://auth0.com/docs/api/management/v2#!/Clients/patch_clients_by_id
+        """
+
+        return self.client.patch(self._url(id, 'enabled_connections', connection_id), data=body)
+
+    def delete_organization_connection(self, id, connection_id):
+        """Deletes a connection from the given organization.
+
+        Args:
+           id (str): Id of organization.
+
+           connection_id (str): the ID of the connection to delete.
+
+        See: https://auth0.com/docs/api/management/v2#!/Clients/delete_clients_by_id
+        """
+
+        return self.client.delete(self._url(id, 'enabled_connections', connection_id))
+
+    # Organization Members
+    def all_organization_members(self, id, page=None, per_page=None):
+        """Retrieves a list of all the organization members.
+
+        Args:
+           id (str): the ID of the organization.
+
+           page (int): The result's page number (zero based). When not set,
+              the default value is up to the server.
+
+           per_page (int, optional): The amount of entries per page. When not set,
+              the default value is up to the server.
+
+        See: https://auth0.com/docs/api/management/v2#!/Clients/get_clients
+        """
+        params = {}
+        params['page'] = page
+        params['per_page'] = per_page
+
+        return self.client.get(self._url(id, 'members'), params=params)
+
+    def create_organization_members(self, id, body):
+        """Adds members to an organization.
+
+        Args:
+           id (str): the ID of the organization.
+
+           body (dict): Attributes from the members to add.
+
+        See: https://auth0.com/docs/api/v2#!/Clients/post_clients
+        """
+
+        return self.client.post(self._url(id, 'members'), data=body)
```

Comment:
just getting ahead of myself, but do you know what the body can take? I tried it from the UI and it was limited to email, app, roles. But what if I want to send some extra metadata to this for "Invite email template"? 
Sorry the docs are still not available so i am trying to read the code. I want to see if we can reuse this instead of custom email from our end inviting user and giving then some links for our app. Links / names are custom to each user so we cannot hardcode it in email template.

Reply:
@aonamrata the template for the "user invitation" email will be customizable. The docs will go live soon and will hopefully help you clarify this. But the [email templates endpoint](https://auth0.com/docs/api/management/v2#!/Email_Templates/patch_email_templates_by_templateName) accepts a type of "user_invitation".